packages/catalog-server/src/lib/firestore/firestore-repository.ts

```diff
@@ -275,31 +290,60 @@ export class FirestoreRepository implements Repository {
   }
 
   async writeCustomElements(
-    packageName: string,
-    version: string,
+    packageVersionMetadata: Version,
     customElements: CustomElementInfo[],
     distTags: string[],
     author: string
   ): Promise<void> {
     // Store custom elements data in subcollection
+    const {name: packageName, version, description} = packageVersionMetadata;
     const versionRef = this.getPackageVersionRef(packageName, version);
     const customElementsRef = versionRef.collection('customElements');
+    const isLatest = distTags.includes('latest');
     const batch = db.batch();
 
+    // Stem the package description
+    const packageDescriptionStems = natural.PorterStemmer.tokenizeAndStem(
+      description ?? ''
+    );
+
     for (const c of customElements) {
+      const tagName = c.export.name;
+      // Grab longer tag name parts for searching. We want "button" from
+      // md-button, etc.
+      const tagNameParts = tagName.split('-').filter((s) => s.length > 3);
```

Comment:
Should we also directly include the un-split tag name as a separate term? It feels like that could naturally boost an exact search someone is trying, but without a "tag:" prefix, like "mwc-button".